packages/react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -135,6 +135,15 @@ export const DatePicker = createComponent<DatePickerOptions>((props, forwardedRe
     targetRef: containerRef,
   });
 
+  // TODO find real solution
+  const overlayPositionProps = {
+    ...overlayProps,
+    style:{
+      ...overlayProps.style,
+      left: isNaN(overlayProps.left) ? 0 : overlayProps.left
+    }
+  }
+
```

Comment:
Out of curiousity - was overlay props defaulting to something that was not undefined or a number?